response.go

```diff
@@ -210,12 +224,14 @@ func (r *Response) Write(w http.ResponseWriter) error {
 	return err
 }
 
+// ResponsesList holds the multiple configured responses.
 type ResponsesList struct {
 	List    map[string]*Response
 	keys    []string
 	current int
 }
 
+// NewResponsesList clears the response list.
```

Comment:
This just returns a new response list

Reply:
You are actually right. 'reset' felt a bit confusing.